### PR TITLE
[DTRA]Ahmad/FEQ-960/Strike Price movement with marker barrier and Price Info

### PR DESCRIPTION
**Pull request by ahmadtaimoor-deriv**
# Description
This card envolves and strkie price movement with marker barrier and setting cross hair to "show price info" on mobile screen

# Cards
https://app.clickup.com/t/86bwhj6y5
 https://app.clickup.com/t/86bwhpu5d

### CODE REVIEW

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

**Review by maryia-deriv — APPROVED** [review]

**Review by balakrishna-deriv — APPROVED** [review]

File: src/store/PriceLineStore.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import EventEmitter from 'event-emitter-es6';
 import { action, computed, observable, when, makeObservable, reaction, IReactionDisposer } from 'mobx';
 import Context from 'src/components/ui/Context';
 import MainStore from '.';
-import { ARROW_HEIGHT, DIRECTIONS, makeElementDraggable } from '../utils';
+import { ARROW_HEIGHT, DIRECTIONS, lerp, makeElementDraggable } from '../utils';
```
Comment (henry-deriv) [inline src/store/PriceLineStore.ts:5]: whats lerp lol

Comment (henry-deriv) [reply src/store/PriceLineStore.ts:5]: could this be changed to something more descriptive

Comment (balakrishna-deriv) [reply src/store/PriceLineStore.ts:5]: lerp is linear interpolation 😄 
https://docs.unity3d.com/ScriptReference/Vector3.Lerp.html

**Review by akmal-deriv — APPROVED** [review]

**Review by henry-deriv — APPROVED** [review]

**Review by balakrishna-deriv — APPROVED** [review]

**Pull request merged**